Formula/mpd.rb

```diff
@@ -27,6 +27,7 @@ class Mpd < Formula
   option "with-libmodplug", "Build with modplug support (for decoding modules supported by MODPlug)"
   option "with-pulseaudio", "Build with PulseAudio support (for sending audio output to a PulseAudio sound server)"
   option "with-upnp", "Build with upnp database plugin support"
+  option "with-libao", "Build with libao support"
```

Comment:
This description is added automatically because you have `depends_on "libao" => :optional` below. So this line can be removed.